docs/motivation.md

```diff
@@ -0,0 +1,532 @@
+# The Verona language: Features and Motivation 
+
+Verona is an object oriented reference capability language with structural
```

Comment:
I think here we should also say which parts of the type system are novel, and which are more standard. 

The capabilities part seems to me not to be fully explained here. Is that so? Is that on purpose? Should there be a reference to OOPSLA?

Reply:
Novelty should be part of the top level description.